fn-fdk.js

```diff
@@ -150,8 +150,8 @@ function convertResult (result, contextout) {
 }
 
 function buildJSONResponse (result, contextout, protoout) {
-  let body = convertResult(result, contextout)
-
+  let body = result !== null ? JSON.stringify(result) : ''
```

Comment:
Couple of issue here: 
* This produces a different behaviour for default contract than JSON (convertResult is there to abstract both paths)  
* The test fail 

convertResult is there to convert the result value into it's rendered output format dependent on the content type - 
the original bug reported here was expected behaviour as the user didn't set the response content type to JSON - this is because the content type defaults to the input content type. 

I think that assumption was probably wrong and it makes sense to default the return content type to JSON (and therefore always try and JSON.stringify by default) 

in `handleDefault`:  
Change
```
 let outCtx = {content_type: ''}
```

to 
```
let outCtx = {content_type: 'application/json`}
```

and in handleJSON: 
```
    let outCtx = {content_type: request.content_type}
```

to 
```
    let outCtx = {content_type: `application/json`}
```

and fix the tests. 

That way we do JSON content by default on output, but a user can still supply a string/html body by changing `context.outputContentType`